tests/tests.js

```diff
@@ -52,6 +53,66 @@ exports.defineAutoTests = function() {
                 done();
             });
         });
+
+        /*
+
+        Tests if the local copy is at the correct place and can be accessed via file plugin.
+        on android this works, as the files are copied to the PERSISTENT location:
+
+         /data/data/<app-id>/files
+
+        on iOS the files are copied to
+
+         /var/mobile/Applications/<UUID>/Library  (or .../Documents in compatibility mode)
+
+        but the persistent directory is .../Library/files.
+
+         */
+        function syncAndTest(appId, success, fail) {
+            var sync = ContentSync.sync({
+                id: appId,
+                type: 'local',
+                copyRootApp: true
+            });
+            sync.on('complete', function(localDataPath) {
+                var file = appId + '/index.html';
+                window.requestFileSystem(LocalFileSystem.PERSISTENT, 0, function(fs) {
```

Comment:
Yeah, but you don't check in failing tests. Then the CI fails. I agree that it should work the same on both iOS and Android and we have issue #83 to track that. That'll be fixed very soon but I'd still rather not to check in a failing test. It just seems wrong to me.